pilot/pkg/features/pilot.go

```diff
@@ -343,7 +343,7 @@ var (
 		"If enabled, Kubernetes services with selectors will select workload entries with matching labels. "+
 			"It is safe to disable it if you are quite sure you don't need this feature").Get()
 	InjectionWebhookConfigName = env.RegisterStringVar("INJECTION_WEBHOOK_CONFIG_NAME", "istio-sidecar-injector",
```

Comment:
do we still need this around if we do the labels?

Reply:
Nope, removed it :+1: